### DISCUSSION TITLE
Tweaks not built in UT that I know of.

### DISCUSSION THREAD

**Discussion by erialsan** [ACGaming/UniversalTweaks#555] [Tweak Suggestions]
This is a list and is actually used in my environment.
Please use it as a reference for future mods with built-in functionality.
Simulated Night
Tips
Item zoom
Durability Tooltip
Advancement Locator
Apple skin
Controlling
~~Ding~~

**Comment by NordicGamerFE** [comment]
While the mod does having something simular to ding the fact that UT ding implemention mutes the ding effect if minecraft volume is muted does not give it full parity to ding
